src/config_db/scripts/sdp_example_workflow.py

```diff
@@ -0,0 +1,40 @@
+
+import ska_sdp_config
+import logging
+logging.basicConfig()
+log = logging.getLogger('main')
```

Comment:
using logger like this is great for now but we will probably need to worry about logger channel namespaces in future so log messages can be properly filtered.